### PR TITLE
DOCS-678 Template Variable Evaluation

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Add Monitor Guide for Template Variable Evaluation

### Motivation
PM Request

### Preview link
https://docs-staging.datadoghq.com/ruth/678/monitors/guide/template-variable-evaluation

### Additional Notes
Updating the notification page is dependent on https://github.com/DataDog/documentation/pull/6524

### CODE REVIEW

File: content/en/monitors/guide/template-variable-evaluation.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Template Variable Evaluation
+kind: guide
+---
+
+In monitor notification messages, you can modify the output of template variables using `eval` syntax, which enables several different mathematical operations and functions on template variables with a numerical value.
```
Comment (mark-azer) [inline content/en/monitors/guide/template-variable-evaluation.md]: using **_the_** `eval` syntax

File: content/en/monitors/guide/template-variable-evaluation.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+| int(var)            | Returns var as an integer                                                                                                                  |
+| float(var)          | Returns var as a float                                                                                                                     |
+| trunc(var)          | Returns var as an integer                                                                                                                  |
+| round(var)          | Returns var rounded to the nearest integer                                                                                                 |
+| ceil(var)           | Returns the ceiling of var (the smallest integer that is greater than or equal to var)                                                     |
+| floor(var)          | Returns the floor of var (the largest integer that is less than or equal to var)                                                           |
+| dec(var)            | Returns the numbers to the right of the decimal point, for example 12.345 returns 0.345.                                                   |
```
Comment (mark-azer) [inline content/en/monitors/guide/template-variable-evaluation.md]: **Change this sentence to:** Returns the numbers to the right of the decimal point, for example if var = 12.345 then dec(var) returns 0.345

File: content/en/monitors/guide/template-variable-evaluation.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+| sgn(var)            | Returns the value of the sign function evaluated at var:<br>sgn(var) = 1 if var > 0<br>sgn(var) = 0 if var = 0<br>sgn(var) = -1 if var < 0 |
+| humanize_bytes(var) | Returns a human readable formatting of var bytes                                                                                           |
+| humanize_bits(var)  | Returns a human readable formatting of var bits                                                                                            |
+| to_bool(var)        | Returns true if var = 1<br>Returns false if var = 0                                                                                        |
+
+### Examples
+
+Use the int function to evaluate the `{{value}}` of a monitor to improve readability and remove the decimals with the following:
```
Comment (mark-azer) [inline content/en/monitors/guide/template-variable-evaluation.md]: **Change this sentence to:** If the the decimal places of the `{{value}}` template variable are unimportant for your particular use cases, use the int function to evaluate `{{value}}` as an integer to improve readability and remove the decimals like so:

File: content/en/monitors/guide/template-variable-evaluation.md (hunk 1 above)
Comment (ruthnaebeck) [inline content/en/monitors/guide/template-variable-evaluation.md]:
```suggestion
In monitor notification messages, you can modify the output of template variables using the `eval` syntax, which enables several different mathematical operations and functions on template variables with a numerical value.
```

File: content/en/monitors/guide/template-variable-evaluation.md (hunk 2 above)
Comment (ruthnaebeck) [inline content/en/monitors/guide/template-variable-evaluation.md]:
```suggestion
| dec(var)            | Returns the numbers to the right of the decimal point, for example if var = 12.345 then dec(var) returns 0.345.                                                  |
```

File: content/en/monitors/guide/template-variable-evaluation.md (hunk 3 above)
Comment (ruthnaebeck) [inline content/en/monitors/guide/template-variable-evaluation.md]:
```suggestion
If the decimal places of the `{{value}}` template variable are unnecessary for your particular use case, use the int function to evaluate `{{value}}` as an integer to improve readability and remove the decimals:
```

**Pull request merged**